src/home/home.html

```diff
@@ -20,7 +20,7 @@
           </div>
         </div>
         <div class="buttonContainer">
-          <div class="launch" click.delegate="navigate('launch')">
+          <div class="launch" click.delegate="navigate('selectPackage')">
```

Comment:
Actually, this change should be reverted.  It is the launch page link that needs to go to selectPackage.

Reply:
Yes true, reverting